### PR TITLE
[Documentation] Adding top nav and collapsible sidebar.

### PR DESCRIPTION
**Pull request by rajkaramchedu**
- Adding a top nav bar.
- Adding a button at bottom left to collapse sidebar for more reading view.
- I am planning to edit the "up and running" doc to add more details, in a checklist format. 

This PR replaces the PR https://github.com/aptos-labs/aptos-core/pull/1190.

### CODE REVIEW

**Comment by rajkaramchedu** [comment]
Why is the Netlify preview cancelled? The Details log shows `6:03:50 PM: Failed during stage 'checking build content for changes': Canceled build due to no content change`. This looks like a new behavior. I need to see deploy preview regardless of whether content changed. @geekflyer?

**Comment by geekflyer** [comment]
> Why is the Netlify preview cancelled? The Details log shows `6:03:50 PM: Failed during stage 'checking build content for changes': Canceled build due to no content change`. This looks like a new behavior. I need to see deploy preview regardless of whether content changed. @geekflyer?

hmm, that's indeed kinda weird. let me check what's going on. I think what happened is probably that between your initial PR and the rebase the content hasn't changed but now you're missing the linking to the previous deployment (before the rebase). I'll look into how to get that notification back.

File: developer-docs-site/docs/guides/up-and-running.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
 ---
-title: "Getting Started"
-slug: "getting-started"
-sidebar_position: 0
+title: "Up and Running"
+slug: "up-and-running"
```
Comment (davidiw) [inline developer-docs-site/docs/guides/up-and-running.md:3]: I want to believe, but "up and running" seems more ambiguous than "getting started"

Comment (rajkaramchedu) [reply developer-docs-site/docs/guides/up-and-running.md:3]: Okay, I was sort of on the fence with it, but now you pushed me over. i will change it.

**Comment by rajkaramchedu** [comment]
Here is the Deploy preview link: https://deploy-preview-1310--aptos-developer-docs.netlify.app/
The above link won't take us there but we can stick in the PR number and it will give us the link, because it is actually building the preview.

**Comment by rajkaramchedu** [comment]
cc: @monica-aptos

**Comment by rajkaramchedu** [comment]
I will have plenty more iterative changes in the docs structure in the coming weeks. For now, this PR only adds the top nav bar.

**Review by davidiw — APPROVED** [review]

**Pull request closed without merging**